### PR TITLE
chore: Add juju commands for using the dashboard.

### PR DESCRIPTION
**Pull request by hatched**
## Done

Added juju commands for using the dashboard to the README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,19 @@ The dashboard to monitor your [Juju](https://juju.is) & [JAAS](https://jaas.ai)
...
 Starting with Juju 2.8 this dashboard is installed by default with every bootstrap. To access it simply run `juju dashboard` and visit the link provided and log in using the supplied credentials. It's also available and automatically updated for users of JAAS on [jaas.ai](jaas.ai/models).
 
+## Using the Dashboard with Juju
+
+Below are the commands you can use to interact with the dashboard with Juju 2.8+
+
+usage: `juju <command>`
+
+| command           | description                                                                                                                            |
+| ----------------- | -------------------------------------------------------------------------------------------------------------------------------------- |
```
Comment (hatched) [inline README.md:14]: This is prettiers doing....

Comment (barrymcgee) [reply README.md:14]: Use Prettier ignore?

File: README.md [hunk 2]
```diff
@@ -4,6 +4,19 @@ The dashboard to monitor your [Juju](https://juju.is) & [JAAS](https://jaas.ai)
 
 Starting with Juju 2.8 this dashboard is installed by default with every bootstrap. To access it simply run `juju dashboard` and visit the link provided and log in using the supplied credentials. It's also available and automatically updated for users of JAAS on [jaas.ai](jaas.ai/models).
 
+## Using the Dashboard with Juju
+
+Below are the commands you can use to interact with the dashboard with Juju 2.8+
```
Comment (barrymcgee) [inline README.md:9]:
```suggestion
Below are the commands you can use to interact with the dashboard from Juju 2.8+
```

File: README.md (hunk 1 above)
Comment (hatched) [reply README.md]: Good idea :) done.

**Pull request merged**